source/renderer/app/stores/WalletsStore.js

```diff
@@ -169,6 +169,14 @@ export default class WalletsStore extends Store {
   @observable transferFundsStep: number = 0;
   @observable transferFundsFee: ?BigNumber = null;
 
+  /* ----------  Hardware Wallet  ---------- */
+  @observable isExportingPublicKeyAborted = false;
+  @observable exportingExtendedPublicKey = false;
+  @observable isDeviceConnected = false;
+  @observable fetchingDevice = false;
+  @observable isTrezor = false;
+  @observable isLedger = false;
```

Comment:
@DeeJayElly this is defined like we will have only one hardware wallet. When we talked about this I meant if you can define those variables inside the container. But ok, I will refine this by applying real logic.